arangodb-net-standard/CursorApi/CursorApiClient.cs

```diff
@@ -77,17 +77,22 @@ public async Task<CursorResponse<T>> PostCursorAsync<T>(PostCursorBody cursorReq
             }
         }
 
+        /// <summary>
+        /// Deletes an existing cursor and frees the resources associated with it.
+        /// DELETE /_api/cursor/{cursor-identifier}
+        /// </summary>
+        /// <param name="cursorId">The id of the cursor to delete.</param>
+        /// <returns></returns>
         public async Task<DeleteCursorResponse> DeleteCursorAsync(string cursorId)
         {
             using (var response = await _client.DeleteAsync(_cursorApiPath + "/" + cursorId))
```

Comment:
We should URL encode `cursorId`

Reply:
#92 is more appropriate in my opinion. These changes are only about adding properties to the response model and I added a comment to the endpoint method.

Reply:
OK I'll merge this then create a new issue if needed